site/content/docs/main/config/request-routing.md

```diff
@@ -283,15 +284,64 @@ spec:
         terminal: true
       - headerHashOptions:
           headerName: User-Agent
+```
+In this example, if a client request contains the `X-Some-Header` header, the value of the header will be hashed and used to route to an upstream Endpoint. This could be used to implement a similar workflow to cookie-based session affinity by passing a consistent value for this header. If it is present, because it is set as a `terminal` hash option, Envoy will not continue on to process to `User-Agent` header or source IP to calculate a hash. If `X-Some-Header` is not present, Envoy will use the `User-Agent` header value to make a routing decision along with the source IP of the client making the request. These policies can be used alone or as shown for an advanced routing decision.
+
```

Comment:
From envoy docs and the actual code behavior here I assume that you cannot ever use multiple hash strategies in one route, therefore docs here were altered to show that only one of `headerHashOptions`, `hashSourceIP` or `queryParameterHashOptions` can be configured. Correct me if I'm wrong.

Reply:
doesn't mean we have to undo what you have here, but might be good to add a little blurb saying they can be used together

Reply:
I'm not sure how it works in the field. I've tried to configure both "header" and "sourceIP" policy for a route and the result was: no hash.

Reply:
hey @pkit had a quick question on this if you had a chance to check it out

sorry for the delay reviewing, everything largely looks good